clusterpools/templates/install-config.aws.yaml.template

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+metadata:
+  name: 
+baseDomain: 
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  replicas: 3
+  platform:
+    aws:
+      rootVolume:
+        iops: 4000
+        size: 100
+        type: io1
+      type: m5.xlarge
+compute:
+- hyperthreading: Enabled
+  name: worker
+  replicas: 3
+  platform:
+    aws:
+      rootVolume:
+        iops: 2000
+        size: 100
+        type: io1
+      type: m5.xlarge
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  aws:
+    region: __CLUSTERPOOL_AWS_REGION__
+pullSecret: "" # skip, hive will inject based on it's secrets
```

Comment:
it's --> its

Reply:
@dhaiducek I don't know how to say it but I stole this install-config from the RHACM product's UI page so we have it wrong in the product!

Reply:
YIKES!!!